From 4eef5cf4d994bfd081a5494463cc792b146bcda2 Mon Sep 17 00:00:00 2001
From: SavantOS Release <actions@users.noreply.github.com>
Date: Mon, 14 Sep 2026 12:51:01 +0000
Subject: [PATCH] Refresh the guest package lock

ca-certificates-mozilla 3.128-1 -> 3.129-1
chromium 152.0.7977.82-1 -> 153.0.8010.36-1
dua-cli 2.44.0-1 -> 2.45.0-1
fzf 0.74.3-1 -> 0.74.4-1
gcr-4 4.4.0.1-1 -> 4.4.1-1
gpu-screen-recorder 6.1.1-1 -> 6.1.2-1
gvfs 1.60.2-4 -> 1.60.3-1
hyprland 0.56.2-2 -> 0.56.2-3
libgcrypt 1.12.3-1 -> 1.12.4-1
libldap 2.6.13-1 -> 2.7.1-1
libportal 0.10.0-1 -> 0.11.0-1
libportal-gtk4 0.10.0-1 -> 0.11.0-1
mkinitcpio 41.1-1 -> 42-1
nss 3.128-1 -> 3.129-1
upower 1.91.3-1 -> 1.91.4-1
atkmm 2.28.5-2 (new)
cairomm 1.14.6-1 (new)
fmt 12.2.0-1 (new)
glibmm 2.66.10-1 (new)
gpsd 3.27.5-1 (new)
gtk-layer-shell 0.10.1-1 (new)
gtkmm3 3.24.11-1 (new)
jsoncpp 1.9.8-1 (new)
libdbusmenu-glib 18.10.20180917-2 (new)
libdbusmenu-gtk3 18.10.20180917-2 (new)
libmpdclient 2.27-1 (new)
libsigc++ 2.12.2-1 (new)
pangomm 2.46.5-1 (new)
playerctl 2.4.1-5 (new)
pps-tools 1.0.3-2 (new)
spdlog 1.17.0-2 (new)
hyprwayland-scanner (removed)
---
 guest/packages.lock.json | 47 ++++++++++++++++++++++++++--------------
 1 file changed, 31 insertions(+), 16 deletions(-)

diff --git a/guest/packages.lock.json b/guest/packages.lock.json
index 5828631..e1864e1 100644
--- a/guest/packages.lock.json
+++ b/guest/packages.lock.json
@@ -16,6 +16,7 @@
     "aquamarine": "0.15.0-2",
     "archlinux-keyring": "20260909-1",
     "at-spi2-core": "2.60.6-1",
+    "atkmm": "2.28.5-2",
     "attr": "2.6.0-1",
     "audit": "4.2.1-1",
     "autoconf": "2.73-1",
@@ -35,11 +36,12 @@
     "bubblewrap": "0.12.0-1",
     "bzip2": "1.0.8-6",
     "ca-certificates": "20240618-1",
-    "ca-certificates-mozilla": "3.128-1",
+    "ca-certificates-mozilla": "3.129-1",
     "ca-certificates-utils": "20240618-1",
     "cairo": "1.18.4-1",
+    "cairomm": "1.14.6-1",
     "cfitsio": "1:4.7.0-1",
-    "chromium": "152.0.7977.82-1",
+    "chromium": "153.0.8010.36-1",
     "clang": "22.1.8-1",
     "compiler-rt": "22.1.8-1",
     "coreutils": "9.11-2",
@@ -60,7 +62,7 @@
     "device-mapper": "2.03.42-1",
     "diffutils": "3.12-2",
     "double-conversion": "3.4.0-1",
-    "dua-cli": "2.44.0-1",
+    "dua-cli": "2.45.0-1",
     "duktape": "2.7.0-7",
     "e2fsprogs": "1.47.4-1",
     "elfutils": "0.196-1",
@@ -83,19 +85,20 @@
     "findutils": "4.11.0-1",
     "flac": "1.5.0-1",
     "flex": "2.6.4-6",
+    "fmt": "12.2.0-1",
     "fontconfig": "2:2.18.3-2",
     "foot": "1.28.0-2",
     "freetype2": "2.14.3-1",
     "fribidi": "1.0.16-2",
     "fuse-common": "3.18.3-1",
     "fuse3": "3.18.3-1",
-    "fzf": "0.74.3-1",
+    "fzf": "0.74.4-1",
     "gawk": "5.4.1-1",
     "gc": "8.2.12-1",
     "gcc": "16.2.1+r23+gd564253eb6c8-1",
     "gcc-libs": "16.2.1+r23+gd564253eb6c8-1",
     "gcr": "3.41.2-2",
-    "gcr-4": "4.4.0.1-1",
+    "gcr-4": "4.4.1-1",
     "gdb": "17.2-1",
     "gdb-common": "17.2-1",
     "gdbm": "1.26-2",
@@ -108,6 +111,7 @@
     "glib-networking": "1:2.80.1-1",
     "glib2": "2.88.3-1",
     "glibc": "2.44+r24+g16be1518495f-1",
+    "glibmm": "2.66.10-1",
     "glslang": "1:1.4.357.0-1",
     "glu": "9.0.3-3",
     "glycin": "2.1.5-2",
@@ -125,7 +129,8 @@
     "gpgme": "2.2.0-1",
     "gpgmepp": "2.2.0-1",
     "gpm": "1.20.7.r38.ge82d1a6-6",
-    "gpu-screen-recorder": "6.1.1-1",
+    "gpsd": "3.27.5-1",
+    "gpu-screen-recorder": "6.1.2-1",
     "graphene": "1.10.8-2",
     "graphite": "1:1.3.15-1",
     "grep": "3.12-2",
@@ -139,16 +144,18 @@
     "gst-plugins-base-libs": "1.28.7-1",
     "gstreamer": "1.28.7-1",
     "gtest": "1.18.0-1",
+    "gtk-layer-shell": "0.10.1-1",
     "gtk-update-icon-cache": "1:4.22.5-1",
     "gtk3": "1:3.24.52-1",
     "gtk4": "1:4.22.5-1",
     "gtk4-layer-shell": "1.3.0-1",
+    "gtkmm3": "3.24.11-1",
     "guile": "3.0.11-1",
     "gum": "2.0.1-1",
     "gupnp": "1:1.6.10-1",
     "gupnp-dlna": "0.12.0-5",
     "gupnp-igd": "1.6.0-2",
-    "gvfs": "1.60.2-4",
+    "gvfs": "1.60.3-1",
     "gzip": "1.14-2",
     "harfbuzz": "14.4.0-1",
     "hdf5": "2.2.0-1",
@@ -161,7 +168,7 @@
     "hyprcursor": "0.1.13-7",
     "hyprgraphics": "0.5.1-4",
     "hypridle": "0.1.8-2",
-    "hyprland": "0.56.2-2",
+    "hyprland": "0.56.2-3",
     "hyprland-guiutils": "0.2.2-3",
     "hyprland-preview-share-picker": "0.2.1-1",
     "hyprlang": "0.6.8-5",
@@ -169,7 +176,6 @@
     "hyprsunset": "0.4.0-3",
     "hyprtoolkit": "0.5.4-6",
     "hyprutils": "0.14.2-1",
-    "hyprwayland-scanner": "0.4.6-1",
     "hyprwire": "0.3.1-3",
     "iana-etc": "20260617-1",
     "icu": "78.3-1",
@@ -189,6 +195,7 @@
     "jq": "1.8.2-1",
     "json-c": "0.19-1",
     "json-glib": "1.10.8-1",
+    "jsoncpp": "1.9.8-1",
     "kbd": "2.10.0-1",
     "keyutils": "1.6.3-4",
     "kmod": "34.2-1",
@@ -236,6 +243,8 @@
     "libcups": "2:2.4.19-1",
     "libdaemon": "0.14-6",
     "libdatrie": "0.2.14-1",
+    "libdbusmenu-glib": "18.10.20180917-2",
+    "libdbusmenu-gtk3": "18.10.20180917-2",
     "libde265": "1.1.2-1",
     "libdecor": "0.2.5-1",
     "libdeflate": "1.26-1",
@@ -259,7 +268,7 @@
     "libfreeaptx": "0.2.2-1",
     "libfyaml": "0.9.6-3",
     "libgcc": "16.2.1+r23+gd564253eb6c8-1",
-    "libgcrypt": "1.12.3-1",
+    "libgcrypt": "1.12.4-1",
     "libgfortran": "16.2.1+r23+gd564253eb6c8-1",
     "libgirepository": "1.86.0-2",
     "libgit2": "1:1.9.7-1",
@@ -286,7 +295,7 @@
     "libksba": "1.8.1-1",
     "liblc3": "1.1.3-2",
     "libldac": "2.0.2.6-1",
-    "libldap": "2.6.13-1",
+    "libldap": "2.7.1-1",
     "liblqr": "0.4.3-1",
     "liblsan": "16.2.1+r23+gd564253eb6c8-1",
     "libluv": "1.52.1-1",
@@ -297,6 +306,7 @@
     "libmnl": "1.0.5-2",
     "libmodplug": "0.8.9.0-7",
     "libmpc": "1.4.1-1",
+    "libmpdclient": "2.27-1",
     "libmysofa": "1.3.5-1",
     "libnautilus-extension": "50.3.1-1",
     "libndp": "1.9-1",
@@ -328,8 +338,8 @@
     "libplacebo": "7.360.1-6",
     "libplist": "2.7.0-3",
     "libpng": "1.6.58-2",
-    "libportal": "0.10.0-1",
-    "libportal-gtk4": "0.10.0-1",
+    "libportal": "0.11.0-1",
+    "libportal-gtk4": "0.11.0-1",
     "libproxy": "0.5.12-1",
     "libpsl": "0.21.5-2",
     "libpulse": "17.0+r98+gb096704c0-1",
@@ -342,6 +352,7 @@
     "libsasl": "2.1.28-5",
     "libseccomp": "2.6.0-1",
     "libsecret": "0.21.7-1",
+    "libsigc++": "2.12.2-1",
     "libsixel": "1.10.5-1",
     "libsm": "1.2.6-1",
     "libsndfile": "1.2.2-4",
@@ -439,7 +450,7 @@
     "mdadm": "4.6-2",
     "mesa": "1:26.2.2-1",
     "minizip": "1:1.3.2-3",
-    "mkinitcpio": "41.1-1",
+    "mkinitcpio": "42-1",
     "mkinitcpio-busybox": "1.36.1-1",
     "mobile-broadband-provider-info": "20251101-1",
     "mpdecimal": "4.0.1-3",
@@ -461,7 +472,7 @@
     "noto-fonts-emoji": "1:2.051-1",
     "npth": "1.8-1",
     "nspr": "4.40-1",
-    "nss": "3.128-1",
+    "nss": "3.129-1",
     "numactl": "2.0.19-1",
     "ocl-icd": "2.3.5-1",
     "omacalc": "0.2.2-1",
@@ -489,6 +500,7 @@
     "pam": "1.7.2-2",
     "pambase": "20260616-1",
     "pango": "1:1.58.2-1",
+    "pangomm": "2.46.5-1",
     "parted": "3.7-1",
     "patch": "2.8-1",
     "pciutils": "3.15.0-1",
@@ -506,12 +518,14 @@
     "pipewire-pulse": "1:1.6.8-1",
     "pixman": "0.46.4-1",
     "pkgconf": "3.0.7-1",
+    "playerctl": "2.4.1-5",
     "plocate": "1.1.25-1",
     "polkit": "127-3",
     "poppler": "26.08.0-1",
     "poppler-glib": "26.08.0-1",
     "popt": "1.19-2",
     "portaudio": "1:19.7.0-4",
+    "pps-tools": "1.0.3-2",
     "procps-ng": "4.0.7-1",
     "psmisc": "23.7-2",
     "pugixml": "1.16-1",
@@ -555,6 +569,7 @@
     "sord": "0.16.22-1",
     "source-highlight": "3.1.9-19",
     "spandsp": "0.0.6-7",
+    "spdlog": "1.17.0-2",
     "speex": "1.2.1-2",
     "speexdsp": "1.2.1-2",
     "spirv-tools": "1:1.4.357.0-1",
@@ -596,7 +611,7 @@
     "udisks2": "2.11.2-1",
     "unibilium": "2.1.4-1",
     "unzip": "6.0-23",
-    "upower": "1.91.3-1",
+    "upower": "1.91.4-1",
     "util-linux": "2.42.3-1",
     "util-linux-libs": "2.42.3-1",
     "uwsm": "0.26.7-1",
-- 
2.55.0

